app/policies/authorizer.rb

```diff
@@ -75,6 +75,10 @@ def creator?
       has_role?(:creator)
     end
 
+    def accesses_mod_response_templates?
```

Comment:
@jeremyf I'm a bit unclear if this is how this class is intended to be used - my prior understanding is that this _ought_ to be a check for `trusted` (and that we _ought_ to be adding `trusted` roles for folks with `admin` or `moderator`), however I found that our test cases in this area often create users with "bare" `admin` or `moderator` roles. I _think_ I want this to live in the `policy`, but we also want to call it in the `Scope`. Right now, this feels like a half-role half-policy hybrid, but maybe there's a better way?